apps/portals/src/LogInDialogContext.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useContext, useState } from 'react'
+
+export type LogInDialogContextType = {
+  setShowLoginDialog: (showLoginDialog: boolean) => void
+  showLoginDialog?: boolean
+}
+
+/*
+ * Stores global state for the "Sign in" dialog in the portals
+ */
+const LogInDialogContext = React.createContext<
+  LogInDialogContextType | undefined
+>(undefined)
```

Comment:
New context to control show/hide login dialog. This was originally state stored in AppInitializer, but moving it to context makes more sense.